app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MerchantsController < ApplicationController
+
+  def index
+    @merchants = MerchantFacade.new
+  end
+
+  def show
+    merchant = MerchantFacade.new
+    items = MerchantFacade.new
+    @merchant = merchant.find_merchant(params[:id])
```

Comment:
You could also call MerchantFacade directly (like: `@merchant = MerchantFacade.new.find_merchant(params[:id])` instead of setting to a variable first. Either way is fine. If you like the idea of setting it first, you could also consider creating a method like:
```ruby
def merchant_facade
  MerchantFacade.new
end
```
because you also make this same call in the index action. Then you could reuse it a bit. The way you have it is fine. I'm just noting other options that you can practice with or that you might see on the job.